capture_hdr_loop.py

```diff
@@ -1,68 +1,65 @@
-"""
-Capture HDR frames in a loop (while actively changing some HDR settings).
+"""Capture HDR frames in a loop (while actively changing some HDR settings).
+
+Three sets of settings are chosen. These settings provide a dynamic range of
+about 9-10 stops, with a decent low- and highlight frame which is sufficient
+to grab good data on most scenes, including shiny for a medium camera operating
+at around 0.7m-1.2m
+
 """
 
 import datetime
+from pathlib import Path
+import yaml
 import zivid
 
 
+def _get_settings(settings: zivid.Settings, path: Path):
+    settings_from_yaml = yaml.load(open(path, "rb").read(), Loader=yaml.Loader)["Settings"]
```

Comment:
The file opened here is actually never closed (`open(path, "rb")`)
Use `path.read_text()` instead.